docs/user-guide.rst

```diff
@@ -290,6 +290,77 @@ Follow the detailed instructions on how to
 .. _Configure Workload Identity Federation from an OIDC identity provider:
     https://cloud.google.com/iam/docs/access-resources-oidc
 
+.. _accessing-resources-using-x509-certificate-sourced-credentials:
+
+Accessing resources using X.509 certificate-sourced credentials
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+For `X.509 certificate-sourced credentials <https://cloud.google.com/iam/docs/workload-identity-federation-with-x509-certificates>`_, the authentication library uses an X.509 certificate and private key to prove your application's identity. The certificate has a built-in expiration date and must be renewed to maintain access.
```

Comment:
you need to follow different format for hyperlinks. see the end of previous section. Something like

.. _OpenID Connect (OIDC):
    https://openid.net/connect/